metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableMetricModule.java

```diff
@@ -123,8 +127,24 @@ public Exposed module(PrimaryComponent primary, Depends depends, String id) {
         @Provides
         @BigtableScope
         public Managed<BigtableConnection> connection(
-            final AsyncFramework async, final ExecutorService executorService
+            final AsyncFramework async, final ExecutorService executorService,
+            final Lazy<FakeBigtableConnection> fakeBigtableConnection
         ) {
+            if (fake.orElse(false)) {
```

Comment:
I'd say, it would be better to have the default value at line 102 where this.fake is being set.